packages/components/src/components/Modal/index.tsx

```diff
@@ -11,13 +11,19 @@ const Header = styled.div`
     margin-bottom: 25px;
 `;
 
-const HeaderBar = styled.div<{ showBottomBorder: boolean }>`
+const HeaderBar = styled.div<{ alignLeft?: boolean; showBottomBorder: boolean }>`
     display: flex;
     align-items: flex-start;
     justify-content: center;
     word-break: break-word;
     padding: 24px;
 
+    ${({ alignLeft }) =>
```

Comment:
For simple css changes I would not use the `css` approach, it can be a one-liner

```
text-align: ${({ alignLeft }) => alignLeft && 'left'};
```

Reply:
I'm, afraid that this one-liner could lead to `text-align:;` in the CSS output.